crates/rover-client/src/query/graph/push.rs

```diff
@@ -45,7 +45,7 @@ fn get_push_response_from_data(
     if let Some(opt_data) = service_data.upload_schema {
         Ok(opt_data)
     } else {
-        Err(RoverClientError::HandleResponse {
+        Err(RoverClientError::AdhocError {
             msg: "No response from mutation. Check your API key & graph name".to_string(),
```

Comment:
Let's hide this implementation detail here. Nobody need'a know we doing graphql :trollface: 
```suggestion
            msg: "No response from the graph registry. Check your API key & graph name".to_string(),
```